### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by katagirya** [brendankelly/micropower#8]
Thank you so much for your tool "micropower" its a great tool.  while running the last command that you provided, > install_github(repo=“micropower”,username=“brendankelly”), i got this error Error: unexpected input in "install_github(repo=�"

i then tried to install the package directly from my computer my downloading the File from GIthub, i got this error;Error in install.packages : cannot install Windows binary packages on this platform
Is there a different package i can install for a mac

**Comment by juyanmei** [comment]
Hi,
You can try to use this code to install the package: install_github("brendankelly/micropower")

**Comment by brendankelly** [comment]
Yes -- with the new devtools syntax, devtools::install_github("brendankelly/micropower") should work. Thanks!
